### PR TITLE
Fix pipeline code not to use docs repository.

### PR DESCRIPTION
**Pull request by ghost**
#### Summary
Fix pipeline code not to use docs repository. That makes fork PRs to fail.

### CODE REVIEW

File: .github/workflows/check-label.yml [hunk 1]
```diff
@@ -12,7 +12,10 @@ jobs:
       if: contains(github.event.pull_request.labels.*.name, 'preview-environment')
       run: echo "label exist, will trigger the preview environment"
     - name: Set PR
-      run: echo ${{ github.event.pull_request.number }} > pr.txt
+      env:
+        PR_NUMBER: ${{ github.event.number }}
```
Comment (pfltdv) [inline .github/workflows/check-label.yml:16]: github.event.pull_request.number is same with github.event.number

File: .github/workflows/preview-env-fork.yml [hunk 2]
```diff
@@ -8,26 +8,21 @@ on:
 jobs:
   deploy:
     runs-on: ubuntu-latest
+    if: ${{ github.event.workflow_run.conclusion == 'success' }}
```
Comment (ghost) [inline .github/workflows/preview-env-fork.yml:11]: It is a common if clause. Moved it to job level.

**Review by jupenur — APPROVED** [review]
LGTM

**Review by phoinixgrr — APPROVED** [review]
:shipit:

**Pull request merged**